taccsite_cms/static/site_cms/css/src/_imports/components/django.cms.blog.plugins.css

```diff
@@ -0,0 +1,15 @@
+/*
+Django CMS Blog Plugin Components
+
+These plugins can be loaded anywhere on the site, not just on blog pages.
+
+Reference:
+
+- [Django CMS Blog Plugin Templates](https://github.com/nephila/djangocms-blog/tree/1.1.1/djangocms_blog/templates/djangocms_blog/plugins)
+
+Styleguide Components.DjangoCMS.Blog.Plugins
+*/
+
+/* No styles yet. Not loaded yet. */
+/* HELP: How to load for entire site, but only if blog is installed? */
+/* IDEA: Use Python setting conditional in assets_site.html to load CSS file */
```

Comment:
**Help.** Please offer your opinion on how to eventually do this. Am I overcomplicating it? (The idea I have is like what I had been doing in `taccsite_cms/templates/assets_site.html`.) _I need not add these styles now, but I must this year for new TACC, and I may later this month for ECEP (who want the Archive plugin)._

Reply:
Are you talking about including TACC and ECEP specific styles using a boolean check in the settings (like the deleted lines 21-23)?

That seems reasonable to me. I can't think of any other flexible way offhand. 

If it was portal specific then I would consider bringing it in with the CMS image in the Custom Resources workflow, but if I am tracking here, it would be available across all portals.

Reply:
**Yes**, [those lines of that file](https://github.com/TACC/Core-CMS/pull/466/files#diff-965ec79d014cafe12b2666a2fde283cd94181aeaeb607aab7dce8bf42d8f5e2d). **No**, not for TACC/ECEPstyles. (The TACC and ECEP mentions were unnecessary specifics describing when and why I need a solution.)

How do I _conditionally_ include Core styles for plugins that can be used anywhere on the site?[^5] (The condition is whether blog is installed.)[^6]

CSS can **not** conditionally, based on Python setting, import the styles into `site.css`[^4], but templates **can** conditionally load stylesheets (like [those lines of that file](https://github.com/TACC/Core-CMS/pull/466/files#diff-965ec79d014cafe12b2666a2fde283cd94181aeaeb607aab7dce8bf42d8f5e2d)).

---

**Stylesheet Table**

↑ means "import into this file".

The question is how to load `app.blog.plugins.css`.

| styles for | `site.css`[^1] |  `app.blog.css`[^2] | `app.blog.plugins.css`[^3] |
| - | - | - | - |
| | load: always, all pages | load: if blog, blog pages | load: if blog, all pages
| CMS App | ↑ | — | — | most pages
| CMS Plugins | ↑ | — | — | could be on any page
| CMS Blog App | — | ↑ | — | only on blog pages
| CMS Blog Plugins | — | — | ↑ | could be on any page


[^1]: Loaded (indirectly) by `templates/base.html`.
[^2]: Loaded directly by `templates/djangocms_blog/base.html`.
[^3]: **Proposal:** Loaded conditionally (and indirectly) by `templates/base.html` (like [those lines of that file](https://github.com/TACC/Core-CMS/pull/466/files#diff-965ec79d014cafe12b2666a2fde283cd94181aeaeb607aab7dce8bf42d8f5e2d)).
[^4]: I could make it a thing, but I think it is not worth the effort.
[^5]: The plugins are provided only by the blog app (to show blog content on non-blog pages).
[^6]: I could ignore the condition and always import blog plugin styles, but it seems bad practice to load styles that will never be used on the non-blog sites.

Reply:
Oh, wow, that's too much Wes. I started brainstorming while replying.

> That seems reasonable to me. I can't think of any other flexible way offhand.

Okay, that's what I'll do then, when I need it, not in this PR.